modules/kde/hm.nix

```diff
@@ -225,7 +225,16 @@ in {
     config.lib.stylix.mkEnableTarget "KDE" true;
 
   config = lib.mkIf (config.stylix.enable && config.stylix.targets.kde.enable && pkgs.stdenv.hostPlatform.isLinux) {
-    home.packages = [ themePackage ];
+    home.packages = [ themePackage pkgs.breeze-icons ];
+
+    qt = {
+      enable = true;
+      style.name = "breeze";
+      platformTheme = "kde";
```

Comment:
The option `qt.platformTheme` has been renamed to `qt.platformTheme.name`.

Reply:
Thanks, was waiting for that to be true in latest nixpkgs stable and I believe that's the case now, so I can finally change this.